slave-dotnet11/Dockerfile.rhel7

```diff
@@ -0,0 +1,30 @@
+FROM openshift3/jenkins-slave-base-rhel7
+
+# Labels consumed by Red Hat build service
+LABEL com.redhat.component="jenkins-slave-dotnet11-rhel7-docker" \
+      name="openshift3/jenkins-slave-dotnet11-rhel7" \
+      version="3.6" \
+      architecture="x86_64" \
+      release="1" \
+      io.k8s.display-name="Jenkins Slave .NET Core 1.1" \
+      io.k8s.description="The jenkins slave dotnet image has the dotnet tools on top of the jenkins slave base image." \
+      io.openshift.tags="openshift,jenkins,slave,dotnet,dotnet11"
+
+ENV BASH_ENV=/usr/local/bin/scl_enable \
+    ENV=/usr/local/bin/scl_enable \
+    PROMPT_COMMAND=". /usr/local/bin/scl_enable"
+
+COPY contrib/bin/scl_enable /usr/local/bin/scl_enable
+
+# Install .NET Core
+RUN yum-config-manager --enable rhel-7-server-dotnet-rpms && \
+    yum-config-manager --disable epel >/dev/null || : && \
+    INSTALL_PKGS="rh-dotnetcore11" && \
```

Comment:
In our [other containers]( https://github.com/redhat-developer/s2i-dotnetcore/blob/master/1.1/Dockerfile.rhel7#L39) we also install `nss_wrapper` and `rh-nodejs4`. For the former you'd need `rhel-7-server-ose-3.2-rpms` for the latter `rhel-server-rhscl-7-rpms`:

     RUN yum install --disablerepo=\* --enablerepo=rhel-7-server-rpms -y yum-utils && \
    yum-config-manager --disable \* && \
    yum-config-manager --enable rhel-server-rhscl-7-rpms && \
    yum-config-manager --enable rhel-7-server-dotnet-rpms && \
    yum-config-manager --enable rhel-7-server-rpms && \
    yum-config-manager --enable rhel-7-server-ose-3.2-rpms && \
    yum install -y --setopt=tsflags=nodocs rh-dotnetcore10 nss_wrapper tar rh-nodejs4-npm && \
    yum clean all && \